src/main/kotlin/dsm/wemeet/global/socket/ChatWebSocketHandler.kt

```
@@ -0,0 +1,60 @@
+            .put("timestamp", result.timestamp)
+
+        receiverSession?.takeIf { it.isOpen }?.sendMessage(TextMessage(responseJson.toString()))
+    }
+
+    private fun userIdFromSession(session: WebSocketSession): String? {
+        val token = session.handshakeHeaders.getFirst("Authorization")
+            ?.removePrefix("Bearer ")?.trim() ?: return null
+
+        return try {
+            jwtProvider.getJws(token).body.subject
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)